concent_api/core/transfer_operations.py

```diff
@@ -203,6 +204,12 @@ def calculate_token_expiration_deadline(
     file_transfer_token.files = files
     file_transfer_token = sign_message(file_transfer_token, settings.CONCENT_PRIVATE_KEY)
 
+    if validate_file_transfer_token(file_transfer_token) is not None:
+        (error_message, error_code) = validate_file_transfer_token(file_transfer_token)
```

Comment:
Don't run this twice, run it once, check the results, then raise exception.

Reply:
Done